  @Test
  public void testFail10() throws Exception {
    setFieldsForTest("testfail10");
    projectExplorer.openItemByPath(pathToCurrentPackage + "/A.java");
    editor.waitActive();
    editor.waitTextIntoEditor(contentFromInA);
    editor.setCursorToLine(13);
    editor.typeTextIntoEditor(Keys.END.toString());
    editor.typeTextIntoEditor(Keys.ARROW_LEFT.toString());
    editor.launchRefactorForm();
    refactor.waitRenameFieldFormIsOpen();
    refactor.waitUpdateReferencesIsSelected();
    refactor.typeAndWaitNewName("g");
    refactor.clickOkButtonRefactorForm();
    askDialog.acceptDialogWithText(
        "Problem in 'A.java'. Another name will shadow access to the renamed element");
    refactor.waitRenameFieldFormIsClosed();
    editor.closeFileByNameWithSaving("A");
  }

